content/english/arduino/8-appendix/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Appendix"
+description: ""
+draft: true
+weight: 1
+translationKey: "arduino-8"
+---
+
+# Appendix
+
+## The components, in detail
+
+Imagine our circuit is a play. Before the final performance in front of an audience, we need a place to practice. This is where the breadboard comes in. 
+> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

```

Comment:
I'm not a huge fan of using these quote indents. Can we either use a bulleted list, or maybe wrap in a notice block?

```suggestion
> > Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

```
```suggestion
{{% notice tip % }}
Breadboards are simple platforms that we can use to build temporary circuits for testing and debugging before finalizing our design and permanently soldering (welding) it together. 

...

{{% / notice % }}
```

Reply:
Fixed with bullet lists